test/lib/propertyGroup.spec.ts

```diff
@@ -0,0 +1,102 @@
+import * as parsers from "../../lib/parsers/index";
```

Comment:
Single quotes in .ts files please. 
If we had eslint + prettier it would probably complain about it 🙂 .

Reply:
Thank you ✅